### PR TITLE
📄 Antitrust Policy [due Oct 27]

### PR DESCRIPTION
**Pull request by ebullient**
[🗳️ Vote progress](https://github.com/commonhaus/foundation/pull/196#issuecomment-2340311888 "IC_kwDOKRPTI86LflNQ")

Revised enough to restart the vote. Scope (persons and activities) should be better defined. 

I removed the emphasis on meetings as we do most of our work asynchronously.

For those that reviewed the first version, look at the second commit for what has changed.

voting group: @commonhaus/cf-egc 

Do one of the following:

- Approve the PR or react with 👍 (:+1:) if it looks good to you
- Review with Comments or react with 👀 (:eyes:) if you're "ok" with it (it may not be your favorite)
- If you think it needs discussion or revision
    - Create a review, add your comments and require changes
    - Use the +- button to make a suggestion (instead of just adding a comment).

### CODE REVIEW

File: policies/antitrust-policy.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Violations of this policy may result in disciplinary action, including suspension or revocation of membership.
+
+## Review and Updates
+
+This policy will be reviewed periodically to ensure it remains effective and relevant. Amendments will be made in accordance with the CF’s governance processes.
+
+## Acknowledgment
+
+All members are required to acknowledge their understanding and acceptance of this policy upon joining CF and at regular intervals as determined by the CF Council.
```
Comment (cealsair) [inline policies/antitrust-policy.md:54]: I suggest adding this acknowledgment to the onboarding process for the CF.

**Review by cealsair — APPROVED** [review]

**Review by velo — APPROVED** [review]

File: policies/antitrust-policy.md (hunk 1 above)
Comment (ebullient) [reply policies/antitrust-policy.md:54]: Once it has been approved, I'll add it to the list of attestations.

File: policies/antitrust-policy.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Commonhaus Foundation Anti-Trust Policy
+
+## Purpose
+
+The Commonhaus Foundation (CF) requires that all members, activities, projects, committees, and working groups comply with applicable anti-trust laws.
```
Comment (gavinking) [inline policies/antitrust-policy.md:5]: Is it "anti-trust"?

Comment (ebullient) [reply policies/antitrust-policy.md:5]: grrr.
```suggestion
The Commonhaus Foundation (CF) requires that all members, activities, projects, committees, and working groups comply with applicable antitrust laws.
```

**Review by evanchooly — APPROVED** [review]

**Comment by ebullient** [comment]
Bump. Moving due date to end of October.

**Comment by ebullient** [comment]
vote::result Proceeding with participation quorum and no issues raised.

**Pull request merged**